### DISCUSSION TITLE
Option to select same weekdays in a month (like all Mondays, .. Fridays)

### DISCUSSION THREAD

**Discussion by scjgit** [gpbl/react-day-picker#1131] [Ideas]
We had requirement like when user selects week day title in the month, it should select same week days in that month.
ie: if user selects a week day title (Friday), it should select all Fridays of that month.

![travel](https://user-images.githubusercontent.com/4884103/49397998-f3a49180-f6f1-11e8-85f3-e09f5811bc24.gif)

Solution:
Pass the Month data as a props to weekday to fetch the same week days from that month

**Comment by voltaaage** [comment]
We have a customized weekday component that allows us to select the weekdays within a given month. The problem we ran into was that weekdays had no context on what month to update and so we were only able to update the weekdays in the first rendered month. Passing the month data down to the weekday component will allow us to get this context.

We're currently using a customized copy of the npm but would like to be able to continue using this library. I imagine this can be used more widely and we would like to request that weekday selection functionality be added to the library. Alternatively, you can allow users of the package to customize the month component as well. 

We can provide code snippets if needed - or submit a PR to make some changes. Let us know what would be preferred.

**Comment by gpbl** [comment]
This is something it would be nice to add to v8.
